fili-core/src/main/java/com/yahoo/bard/webservice/table/MetricUnionCompositeTable.java

```diff
@@ -78,4 +79,40 @@ public MetricUnionCompositeTable(
                 )
         );
     }
+
+    /**
+     * Constructor.
+     *
+     * @param name  Name that represents set of fact table names joined together
+     * @param timeGrain  The time grain of the table. The time grain has to satisfy all grains of the tables
+     * @param columns  The columns for this table
+     * @param physicalTables  A set of PhysicalTables that are put together under this table. The tables shall have
+     * zoned time grains that all satisfy the provided timeGrain
+     * @param logicalToPhysicalColumnNames  Mappings from logical to physical names
+     * @param representativeAvailability  A single source availability that represents the coalesced data availability
+     * of this composite table.
+     * @param availabilitiesToMetricNames  A map of all availabilities to set of metric names
+     */
+    public MetricUnionCompositeTable(
+            @NotNull TableName name,
+            @NotNull ZonedTimeGrain timeGrain,
+            @NotNull Set<Column> columns,
+            @NotNull Set<ConfigPhysicalTable> physicalTables,
+            @NotNull Map<String, String> logicalToPhysicalColumnNames,
+            @NotNull Availability representativeAvailability,
```

Comment:
There are actually two options here

1. Pass in a single composite `Availability`, or
2. Pass in a collection of base `Availabilities`

I choose the first option for more concise implementation. But I'm open for any changes